changelog/unreleased/10552

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix crash in Linux AppImage updater
+
+In between the releases 3.0.0 and 3.1.0, a dependency was updated, causing an ABI
```

Comment:
```suggestion
In between the releases 3.0.0 and 3.1.0, a dependency was updated, causing an API
```

Reply:
It's an ABI change, not an API change. The Application Binary Interface was incompatible.